exchanges/binance/binance_test.go

```diff
@@ -70,23 +68,29 @@ func TestGetRecentTrades(t *testing.T) {
 }
 
 func TestGetHistoricalTrades(t *testing.T) {
-	t.Parallel()
-	_, err := b.GetHistoricalTrades("BTCUSDT", 5, 1337)
-	if err == nil {
+	TestSetup(t)
+
+	_, err := b.GetHistoricalTrades("BTCUSDT", 5, 0)
+	if !mockTests && err == nil {
+		t.Error("Test Failed - Binance GetHistoricalTrades() expecting error")
+	}
+	if mockTests && err == nil {
```

Comment:
Nope because I had to drop support for the function as it was always returning {"code":-2014,"msg":"API-key format invalid."} All other functions seemed to work. Needs an update on API.